### PR TITLE
Implement Weather UI & Logic

### PR DESCRIPTION
**Pull request by Yannic-reust**
Done ToDo's:
- fetch api data
- ui & components for weather
- sort api data
- display api data

### CODE REVIEW

**Comment by Yannic-reust** [comment]
Read for Review @DerBeton

File: src/components/WeatherTodayDetail/WeatherTodayDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "Schnee Tal",
+    "Lawinengefahrenstufe",
+  ];
+  return (
+    <>
+      <div>
+        {elements.map((element, index) => (
+          <div key={index} className="flex place-content-between ">
+            <p>{element}</p>
+            <p className="font-semibold">asd</p>
```
Comment (Cyril676) [inline src/components/WeatherTodayDetail/WeatherTodayDetail.jsx:16]: Just a reminder to remove or change the "asd"

File: src/components/WeatherTile/useWeatherCode.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const useWeatherCode = (weatherCode) => {
+  const ICON_CLASSES = "mt-6 fill-white w-10";
+
+  const [icon, setIcon] = useState(null);
+
+  useEffect(() => {
+    let weatherCondition = null;
+    switch (weatherCode) {
+      case 0:
+        weatherCondition = <SunnySVG className={ICON_CLASSES} />;
```
Comment (Cyril676) [inline src/components/WeatherTile/useWeatherCode.jsx:19]: SVG elements should have an alt tag just like Image elements describing what they are

**Review by Cyril676 — COMMENTED** [review]
I have not found any major changes to your code. Apart from a few comments to remove and that you should add an alt tag to the svg's.

**Comment by Yannic-reust** [comment]
fixed all requests

**Pull request merged**